internal/handlers/register_test.go

```diff
@@ -0,0 +1,58 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/magiconair/properties/assert"
```

Comment:
I don't think we need new library (for properties handling) just to run assert statements in tests

Reply:
GoLand auto imported this dep. :rofl: Will fix :+1: